server/test/medium/specs/services/workflow.service.spec.ts

```diff
@@ -611,6 +611,52 @@ describe(WorkflowService.name, () => {
         sut.update(auth, created.id, { actions: [{ actionId: factory.uuid(), actionConfig: {} }] }),
       ).rejects.toThrow();
     });
+
+    it('should update trigger type', async () => {
```

Comment:
this should be a unit test, or you should assert against the database after the update.